docs/dqx/docs/guide/quality_checks.mdx

```diff
@@ -713,3 +713,144 @@ The below example demonstrates how to extract the results from a reporting colum
 
 An example of how to provide user metadata can be found in the [Additional Configuration](/docs/guide/additional_configuration) section.
 
+## Applying Checks and Writing to Delta Tables
```

Comment:
Do we need to create a new section. Can we add it to the existing examples?

as alternative to 
`apply_checks` -> `save_results_in_table`